plugins/mergePaths.js

```diff
@@ -10,6 +13,19 @@ import { path2js, js2path, intersects } from './_path.js';
 export const name = 'mergePaths';
 export const description = 'merges multiple paths in one if possible';
 
+/**
+ * @param {XastElement} element
+ * @param {ComputedStyles} computedStyle
+ * @param {string} attName
+ */
+function elementHasURL(element, computedStyle, attName) {
+  const style = computedStyle[attName];
+  if (!style || style.type !== 'static') {
+    return false;
+  }
+  return style.value && style.value.includes('url(');
```

Comment:
Perhaps this should be refactored to reuse the `includeUrlReference` utility?

Just helps to keep how we determine if something has a URL reference consistent. I'd also propose using `Url` instead of `URL` for consistency as well, for example when compared to our existing `includesUrlReference` function.

```suggestion
 * @param {ComputedStyles} computedStyle
 * @param {string} attName
 * @returns {boolean}
 */
function elementHasUrl(computedStyle, attName) {
  const style = computedStyle[attName];

  if (style?.type === 'static') {
    return includesUrlReference(style.value);
  }

  return false;
```

(This would also require changing other references to `elementHasURL`.)